server/models/user_model.py

```diff
@@ -1,4 +1,4 @@
-from server.run import db
+from server.extensions import db
```

Comment:
Can we run the server from the server folder, meaning we would remove __init__.py in the server folder and remove all references to server (instead it should be `from extensions import db` , etc.)